src/routes/api/google-conversion/+server.ts

```
@@ -202,6 +248,81 @@ function parseTestimonialSection(content: any[]) {
+ */
+function parseTeamMemberSection(content: any[]): TeamMemberBlock | null {
+  if (content.length >= 5 && (content[0] as any[]).length === 2) {
+    const contentHead = content[0];
+
+    if (
+      contentHead[0][0]?.type === 'paragraph' &&
+      contentHead[0][0]?.data?.text === 'type' &&
+      contentHead[1][0]?.type === 'paragraph' &&
+      contentHead[1][0]?.data?.text === 'teamMember'
+    ) {
+      let currentMember: Record<string, string> = {};
```

ESLint

[error] 'currentMember' is never reassigned. Use 'const' instead.

(prefer-const)